PITCHME.css

```
@@ -1,6 +1,4 @@
-code {
-  background-color: #EEEEEE;
-  color: red;
-  display: inline-block;
+.reveal pre {
+  width:96%;
```

```suggestion
  width: 96%;
```
Just to keep things consistent